Makefile

```diff
@@ -121,7 +121,7 @@ fmt:
 	go fmt ./...
 
 test:
-	CGO_ENABLED=0 go test -v -failfast ./awscsm/... ./cfg/... ./cmd/... ./handlers/... ./internal/... ./logger/... ./logs/... ./metric/... ./plugins/... ./profiler/... ./tool/... ./translator/...
+	CGO_ENABLED=0 go test -coverprofile coverage.txt -failfast ./awscsm/... ./cfg/... ./cmd/... ./handlers/... ./internal/... ./logger/... ./logs/... ./metric/... ./plugins/... ./profiler/... ./tool/... ./translator/...
```

Comment:
I'm not sure if unit test cache will work after this change. The build will still be cached so I do not think it will matter too much. @SaxyPandaBear are unit tests cached currently in this repo?

Reply:
I don't think it cached tests before either. Pretty sure it only cached build artifacts but I didn't pay too much attention

Reply:
Test cache can work without -v or are you referencing the extra coverage addition?